src/components/Account/__snapshots__/Account.test.tsx.snap

```diff
@@ -8,14 +8,19 @@ exports[`<Account> > renders and matches snapshot 1`] = `
     <div
       class="header-class"
     >
-      <h3>
+      <h2>
         account.about_you
-      </h3>
+      </h2>
     </div>
     <form
       class="_flexBox_1c1c63"
       novalidate=""
     >
+      <h1
+        class="hideUntilFocus"
+      >
+        nav.account
+      </h1>
```

Comment:
This is kind of a strange snapshot. Can it happen that an h2 appears first, before the h1? Is the h1 perhaps placed incorrectly?

Reply:
Ah yes. So then it is correct? It is not rendered as such in the DOM? (An Axetool check would notice it)

Reply:
Yes it is :)